core/email_api.php

```diff
@@ -1144,7 +1144,9 @@ function email_send( EmailData $p_email_data ) {
 	$t_mail->LE         = "\r\n";
 	$t_mail->Encoding   = 'quoted-printable';
 
-	if( !empty( $t_debug_email ) ) {
+	# String '' or value OFF (backward compatibility with old configs) are considered as debugging
+	# is not enabled.
+	if( !empty( $t_debug_email ) && $t_debug_email !== OFF ) {
```

Comment:
As per [my note ~52738](https://www.mantisbt.org/bugs/view.php?id=20684#c52738), this change is really not necessary, since in PHP, `'' == 0`.

The first predicate will be only be true when $t_debug_email != [0, '' or false], so the 2nd is either not evaluated or true.

Suggest revert to original code.